contracts/staking/IApplication.sol

```diff
@@ -24,7 +24,9 @@ pragma solidity 0.8.9;
 interface IApplication {
     /// @notice Used by T staking contract to inform the application that the
     ///         authorized amount for the given staking provider increased.
-    ///         The application may do any necessary housekeeping.
+    ///         The application may do any necessary housekeeping. The
+    ///         application may revert the transaction in case the authorization
```

Comment:
```suggestion
    ///         application must revert the transaction in case the authorization
```
Otherwise, it doesn't make much sense to define it as a "minimum authorization"

Reply:
Makes sense! Updated in the last commit.

Reply:
Code format is complaining but I'll fix it once #87 is merged when rebasing over `main`. Ignoring for now.